src/features/ProductAvailability/ProductAvailability.tsx

```diff
@@ -5,14 +5,25 @@ import styles from './ProductAvailability.module.scss'
 /**
  * Компонент с версткой информации о наличии товара на складе и его артикулом.
  */
-export const ProductAvailability: FC = () => {
+type Props = {
+  code: number
+  quantity: number
+}
+export const ProductAvailability: FC<Props> = ({ code, quantity }) => {
   return (
     <div className={styles['product-card__info']}>
       <span className={styles['product-card__availability']}>
-        <span className={styles['product-card__status-dot']}></span>
-        На складе
+        {quantity > 0 ? (
+          <>
+            <span className={styles['product-card__status-dot']}></span>На складе
+          </>
+        ) : (
+          <>
+            <span className={styles['product-card__status-zero']}></span>Нет на складе
```

Comment:
Насколько вижу, здесь самозакрывающийся тег, если мы не передаем ничего в тег, то мы его сразу закрываем.
```suggestion
            <span className={styles['product-card__status-zero']} />Нет на складе
```